src/pages/billing/List.js

```diff
@@ -142,7 +142,7 @@ const ProjectListItem = ({ project, project: { roles, status, cloudPlatform }, l
           ]),
           //Currently, only billing projects that failed to create can have actions performed on them.
           //If that changes in the future, this should be moved elsewhere
-          h(BillingProjectActions, { project, loadProjects })
+          isOwner && h(BillingProjectActions, { project, loadProjects })
```

Comment:
FYI @MatthewBemis I believe you added this menu-- if you try to delete the project as a non-owner you will get an error. Just tagging you in case there was some reason why you felt like it should always be displayed.

Reply:
Ah, I didn't think to protect against this case because I didn't believe it was possible to share a failed billing project. My mistake, and thanks for fixing!

Reply:
No problem, it may not be possible for GCP projects. For Azure, a cause of the error state can be if the user does not have permissions on the associated BPM record (which shouldn't be a state you can easily get into once the Rawls PR is merged).